instrumentation/graphql_batch/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Have we (Shopify) contributed Apache licenses code in the past?

Reply:
yep, Shopify has contributed to this repo earlier.